cli/users.go

```diff
@@ -117,23 +123,22 @@ var cmdUsers = []cobra.Command{
 
 			logOK()
 		},
-	},
-}
+	}
 
-// NewUsersCmd returns users command.
-func NewUsersCmd() *cobra.Command {
 	cmd := cobra.Command{
 		Use:   "users",
 		Short: "Users management",
 		Long:  `Users management: create accounts and tokens"`,
 		Run: func(cmd *cobra.Command, args []string) {
-			logUsage("Usage: users [create | get | update | token | password]")
+			logUsage("users [create | get | update | token | password]")
 		},
 	}
 
-	for i := range cmdUsers {
-		cmd.AddCommand(&cmdUsers[i])
-	}
+	cmd.AddCommand(&createCmd)
```

Comment:
Adding command by cmmand is maybe good for documentation purposes, but I kinda liked this geeky loop.

@dusanb94 @manuio @nmarcetic opnions?

Reply:
I'm OK with either solution, I really don't have preferences.

Reply:
It's not possible to add flags easily when the commands are in the array, this is why I changed it.

Reply:
If we do this here we will have to do it for all commands and this should be done inn a separate PR and discussed before. I also prefer to geeky loop :)